src/actions/user_actions.js

```diff
@@ -33,9 +34,9 @@ export const editUser = (boolean) => {
   }
 }
 
-export const updateSettings = (userId, adminStatus) => {
+export const updateSettings = (userId, adminStatus, callback) => {
   let switchAdmin;
-    adminStatus == true ? switchAdmin = false : switchAdmin = true;
+    switchAdmin =  adminStatus == true ? false : true;
```

Comment:
I think you could just delete both of these and use `{ "admin": !adminStatus }` on line 45 instead.